### DISCUSSION TITLE
Output report URL as comments to a PR

### DISCUSSION THREAD

**Discussion by debdutgoswami** [GoogleChrome/lighthouse-ci#658] [Q&A]
Is there any way of providing the report URL in a PR comments?

I have setup CI to run to every PR to main so I would be best to have the report url in the comments of that PR.

**Comment by patrickhulce** [comment]
Not out-of-the-box, but you might be interested in using the output of `links.json` in some way to make the links more accessible anywhere you'd like :)

**Reply by debdutgoswami** [reply]
Okay got it. Seems like I need to me more creative with this.
